docs/index.md

```diff
@@ -128,10 +127,13 @@ options for deployment:
 * [Cluster Overview](cluster-overview.html): overview of concepts and components when running on a cluster
 * [Submitting Applications](submitting-applications.html): packaging and deploying applications
 * Deployment modes:
-  * [Amazon EC2](https://github.com/amplab/spark-ec2): scripts that let you launch a cluster on EC2 in about 5 minutes
```

Comment:
I moved this to the end of this list because the link is archived on 2017.

<img width="596" alt="Screenshot 2024-09-13 at 15 59 26" src="https://github.com/user-attachments/assets/6b62f965-c16b-485c-b3d8-7a747c66bae7">